docs/source/server_setup.rst

```diff
@@ -33,14 +33,18 @@ Required environment variables
 ------------------------------
 
 The following environment variables will need to be set when running HAWC in
-production; example values are shown for each:
+production.
+ - For linux, you can set these values in ``$VIRTUAL_ENV/bin/postactivate``.
```

Comment:
``postactivate`` only exists when using virtualenvwrapper I think; i think it should just be ``activate``.